phpcs.xml.dist

```diff
@@ -18,6 +18,7 @@
         <exclude name="Generic.Formatting.MultipleStatementAlignment"/>
         <exclude name="Squiz.Commenting.FunctionComment.SpacingAfterParamType"/>
         <exclude name="SlevomatCodingStandard.Functions.StaticClosure"/>
+        <exclude name="SlevomatCodingStandard.ControlStructures.RequireNullCoalesceEqualOperator.RequiredNullCoalesceEqualOperator"/>
```

Comment:
Do we want to support PHP 7.3 still? It EOLs in few months (6 Dec 2021)

Reply:
Good point, I totally agree.